interface/ceed-basis.c

```diff
@@ -605,7 +572,7 @@ int CeedMatrixMatrixMultiply(Ceed ceed, const CeedScalar *mat_A, const CeedScala
       CeedScalar sum = 0;
 
       for (CeedInt k = 0; k < kk; k++) sum += mat_A[k + i * kk] * mat_B[j + k * n];
-      mat_C[j + i * n] = sum;
+      mat_C[j + i * n] = (fabs(sum) < 5 * CEED_EPSILON) ? 0.0 : sum;
```

Comment:
I'm not sure if we should mess around with how every matrix-matrix product works to fix one flakey test?

Reply:
Also, 5*eps I don't think was enough for me locally.

Reply:
Yeah, I was thinking to avoid an extra for loop for rounding in `CeedBasisCreateProjectionMatrices`. We can try that.

Reply:
The projection creation is really infrequent, so we don't have to worry about high efficiency there.

Reply:
Same error. Can we change the tolerance for that example?

Reply:
The GPU job hasn't finished yet. The rust error is something else that I'm looking at

Reply:
Yeah, the last commit is good. The rust issue is unrelated